### DISCUSSION TITLE
Some shortcuts for selecting assistants

### DISCUSSION THREAD

**Discussion by antonme** [danny-avila/LibreChat#62] [Feature Requests & Suggestions]
What do you think of adding some simple shortcuts for selecting assistants? Like Cmd+Number, for example.

**Comment by danny-avila** [comment]
> Currently, it's not possible to select an assistant using only the keyboard. Instead, you have to click on the assistant icon, which may not be > obvious to all users. My proposal is to make the assistant menu appear when the user presses the "return" key without typing anything in > the input field. This would make it easier to choose an assistant without needing to use the mouse, and it may also be more discoverable than having to click on the assistant icon manually

Excellent ideas, the cmd + # as well. I'll plan on implementing this

**Comment by wtlyu** [comment]
Why not try Shift + Tab, then press Enter. It's already functional

**Reply by antonme** [reply]
> Why not try Shift + Tab, then press Enter. It's already functional

It is even harder to discover than clicking on the icon by itself :)
At the same time, pressing return with an empty prompt doing absolutely nothing right now

**Reply by wtlyu** [reply]
aha it's true.

Let's imagine more interesting ways, eg just tell gpt that you want to change the model :) That sounds cool

**Reply by antonme** [reply]
That does sounds cool, but that would force us to add something to the prompt of each mode, which is not really that good, so I am all for some simple keyboard controls :)

**Comment by wtlyu** [comment]
I suggest to design a complete keybinding schema. Not only for model select. but also regenerate, cancell, show or close of nav...
